### DISCUSSION TITLE
New Command: php artisan module:make-component ComponentName ModuleName

### DISCUSSION THREAD

**Discussion by aaqibmehran** [nWidart/laravel-modules#1498] [Ideas]
I think a new command to add blade components should be added in this package.
Just like default Laravel blade components in https://laravel.com/docs/9.x/blade#components.

`php artisan module:make-component ComponentName ModuleName`

This command should generate two files:
`Modules\ModuleName\View\Components\ComponentName.php`
`Modules\ModuleName\resources\views\components\component-name.php`

**Comment by dcblogdev** [comment]
this is now in place

![CleanShot 2023-02-24 at 13 31 12](https://user-images.githubusercontent.com/1018170/221190885-ead37b0c-3a34-4c91-bd3a-98055a462b6c.png)

**Reply by aaqibmehran** [reply]
In which version @dcblogdev ? I just upgraded to v10 and these commands are not working
